software/bedtools/complement/meta.yml

```diff
@@ -0,0 +1,30 @@
+name: bedtools complement
+description: Returns all intervals in a genome that are not covered by at least one interval in the input BED/GFF/VCF file
+keywords:
+    - complement
+tools:
+    - bedtools:
+        description: |
+            Bedtools is a software package that provides with a toolset to perform genome arithmetic operations.
+        homepage: https://bedtools.readthedocs.io/en/latest/index.html
+        documentation: https://bedtools.readthedocs.io/en/latest/index.html
+        doi: 10.093/bioinformatics/btq033
+input:
+    -
```

Comment:
Again, a general comment: Does this really need to be a nested list? 
Even if the inputs are grouped, they all have a unique name. 

Or maybe we could introduce docstrings? It would be 
nice to have code and docs more closely together.

Just thinking out loud... not specific to this review and for a later iteration anyway...